### DISCUSSION TITLE
How to get latest update project ?

### DISCUSSION THREAD

**Discussion by Vikash861** [EddieHubCommunity/LinkFree#6362] [General]
how to get the latest project in my forked repo and local environment.

**Comment by kumarsonsoff3** [comment]
To get the latest updated project to your fork, you need to `Sync` it with the main branch using the <kbd>Sync</kbd> button in your fork. 

Then you can pull the changes to your local environment using `git`.

Something similar to this!
![IMG_20230503_092255](https://user-images.githubusercontent.com/88786642/235830121-e0ae5a3b-1f5d-4765-b204-9108f3a4dd77.jpg)

You might want to look at these docs & video: https://linkfree.io/docs/how-to-guides/editing
